testbed/correctness/metrics/metric_index.go

```diff
@@ -0,0 +1,50 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Same as above. Should be in testbed in form that can be used in contrib project.